cmake/find/FindNASM.cmake

```diff
@@ -0,0 +1,35 @@
+# This module will set `NASM_FOUND` if NASM is found.
+# If NASM is found, `NASM_ROOT` will be set to the root folder.
+# `NASM_BIN_DIR` will be set to the folder containing the executable,
+# and `NASM_EXECUTABLE` will be set to the path of the NASM executable.
+
+
+if(HUNTER_STATUS_DEBUG)
+  message("[hunter] Custom FindNASM module")
+endif()
+
+# Try to find it even if it may already be set.
+find_path(
+  NASM_ROOT
+  NAMES
+  bin/nasm
+  bin/nasm.exe
+  )
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(
+  NASM
+  DEFAULT_MSG
+  NASM_ROOT
+  )
+
+if(NASM_FOUND)
```

Comment:
It should be always `TRUE`, right? Probably if it's not we should return `FATAL_ERROR`.